lib/Migration/Version4200Date20241210000000.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Mail\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version4200Date20241210000000 extends SimpleMigrationStep {
+
+	/**
+	 * @param IOutput $output
+	 * @param Closure(): ISchemaWrapper $schemaClosure
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		$schema = $schemaClosure();
+
+		$outboxTable = $schema->getTable('mail_local_messages');
+		if ($outboxTable->hasColumn('body')) {
+			$outboxTable->dropColumn('body');
```

Comment:
What will be the migration path for production instances?

Reply:
I did that before I realized that we use the same table for drafts, so at the time didn't think a migration was necessary.

Now, going to rename instead of dropping, then execute a sql command postSchemaChange to move the html versions to the body_html column based on the Html flag. Unless there is anything wrong with that?